atomicapp/providers/openshift.py

```diff
@@ -34,11 +34,13 @@ def init(self):
 
         if "openshiftconfig" in self.config:
             self.config_file = self.config["openshiftconfig"]
+            if self.container:
+                self.config_file = "/host" + self.config_file
         else:
             logger.warning("Configuration option 'openshiftconfig' not found")
 
         if not self.config_file or not os.access(self.config_file, os.R_OK):
-            raise ProviderFailedException("Cannot access configuration file %s" % self.config_file)
+            raise ProviderFailedException("Cannot access configuration file %s. Try adding 'openshiftconfig = /path/to/your/.kube/config' in the [general] section of the answers.conf file." % self.config_file)
```

Comment:
@vpavlin could you take this one line into https://github.com/projectatomic/atomicapp/pull/213 so we can close this PR due to dup?